designer/server/src/test/scala/pl/touk/nussknacker/ui/api/AppApiHttpServiceBusinessSpec.scala

```diff
@@ -120,8 +116,8 @@ class AppApiHttpServiceBusinessSpec
 
           MockableDeploymentManager.configureScenarioStatuses(
             Map(
-              "id1" -> SimpleStateStatus.Running,
-              "id2" -> SimpleStateStatus.Running,
+              "id1" -> BasicStatusDetails(SimpleStateStatus.Running, Some(VersionId(1))),
```

Comment:
Now, we need to mock at least version, because tests test wider context of scenario status resolution which check if deployed version matches requested deploy action.